sgkit/display.py

```diff
@@ -129,7 +129,7 @@ def truncate(ds: xr.Dataset, max_sizes: Mapping[Hashable, int]) -> xr.Dataset:
 
 def display_genotypes(
     ds: xr.Dataset, max_variants: int = 60, max_samples: int = 10
-) -> GenotypeDisplay:
+) -> object:
```

Comment:
I made this change because `GenotypeDisplay` isn't public (and probably doesn't need to be), and Sphinx was complaining about that. There may be a better way of dealing with this though.

Reply:
Nevermind, I changed this back as I found a way of getting Sphinx to ignore this (while also keeping mypy happy).